deploy-board/deploy_board/webapp/env_views.py

```diff
@@ -503,6 +513,19 @@ def get(self, request, name, stage=None):
         return response
 
 
+def _getLastClusterRefreshStatus(request, env):
+    try:
+        replace_summaries = clusters_helper.get_cluster_replacement_status(
+            request, data={"clusterName": '{}-{}'.format(env["envName"], env["stageName"])})
+
+        if len(replace_summaries["clusterRollingUpdateStatuses"]) == 0:
```

Comment:
any case replace_summaries can be None or it doesn't have key clusterRollingUpdateStatuses